CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ agree to abide by its terms. To report violations, send an email to [patternfly@
 
 ## Issues and Project Board
 
-We use issues to track work items, such as bug reports and feature requests. Issues can be found in the [issue tracker](https://github.com/patternfly/patternfly-react/issues) or [project board](https://github.com/patternfly/patternfly-react/projects/1). We use the project board to help visualize and manage status of an issue, and we use labels to help prioritize and identify issues.
+We use issues to track work items, such as bug reports and feature requests. Issues can be found in the [issue tracker](https://github.com/patternfly/patternfly-react/issues) or on our [Zenhub project board](https://github.com/patternfly/patternfly-react/issues#workspaces/pf4-workspace-5b2142ff9499cb7cdaf1e632/board?repos=84372347). The [Zenhub extension](https://www.zenhub.com/extension) is required to view the project board. We use the project board to help visualize and manage status of an issue, and we use labels to help prioritize and identify issues.
```

Comment:
We can have zenhub links here.  Not everyone has it installed.

Reply:
We should have a tutorial for how to install Zenhub.